### DISCUSSION TITLE
Lists bug

### DISCUSSION THREAD

**Discussion by Neko7sora** [community/community#7502] [Mobile] [answered]
If the language is other than English, it will not respond.

https://user-images.githubusercontent.com/75793267/140908126-c5a45775-4ece-4cc8-a784-6a07b9112694.MP4

**Comment by brianlovin** [comment, marked as answer]
Thanks for the report @Neko7sora — I've shared this with our team, and we've already prepared a fix that will be live in the next production version of the app!
